addon/components/frost-action-bar.js

```diff
@@ -57,7 +65,18 @@ export default Component.extend({
           return `${count} ${items} selected`
         }
       },
-      isLoading: false
+      isLoading: false,
+
+      @readOnly
+      @computed('selectedItems.[]')
```

Comment:
Several things about this:

1) There is already an `isVisible` property on Ember Components so you should not name yours the same - https://www.emberjs.com/api/ember/2.18/classes/Component/properties/isVisible?anchor=isVisible

2) Because this computed property is `@readOnly` you will not be able to override it by passing it in via the `isVisible` property in the component template usage as believed.

3)  I am pretty certain that `ember-prop-types` will not set this computed property up as desired since it is included as part of `getDefaultProps()`

4) This approach is not a Ciena-supported one.  We do not want to support the ability to override a computed property definition, which is why we use `@readOnly`.  What should be done instead is support a property that can be used to toggle visibility.  Then you will still have a computed property that will take the existence and value of that property into account within its own definition, as well as the `selectedItems.[]` content.

Reply:
It seems there's no test that is testing the scenario you desired?

Reply:
This is something I wanted to ask you about, but you were gone by time I got to it on Friday. So, this does work, I believe because of the way getDefaultProps works--but not really a great approach. I didn't want to use a different prop because it seemed strange to have two props for its visibility. But, isVisible isn't part of the interface... which for some reason wasn't occurring to me at the time. I'm going to blame this on it being Friday at the time. I think I'll just move it to "forceVisibility" and use that in the isVisible computed prop.

Reply:
And add a test for it.